src/code/utilities/url-params.js

```diff
@@ -12,4 +12,16 @@ var urlParams;
        urlParams[decode(match[1])] = decode(match[2]);
 })();
 
+export function getClassId() {
+  // Pull the class number out of the class info
+  let regex = /classes\/(\d*)/g,
+      matches = regex.exec(urlParams.class_info_url);
+
+  return matches ? matches[1] : null;
+}
+
+export function getUserId() {
+  return urlParams.domain_uid;
```

Comment:
Good questions! As for `classes`, I just signed up for 2 and it still seems to send just the desired class ID. I'm still not sure why they're named what they are, would you be able to help us out @scytacki?